### DISCUSSION TITLE
[Firebase_Auth - Flutter] SMS verification code request failed: unknown status code: 17499 Internal error encountered.

### DISCUSSION THREAD

**Discussion by MSatyam-Mishra** [firebase/flutterfire#10789] [Q&A]
i am getthing  **SMS verification code request failed: unknown status code: 17499 Internal error encountered.** error on my Flutter app while sending OTP. 
this code was working perfectly 2 days ago, but yesterday when tried this, it didn't work. i thought I messed configuration so I created a new flutter project and did it's firebase configuration from the beginning but getting same error.

### My Code is :

sendOTP() async {
    String phoneNumber = countryCodeController.text + phone;

    try {
      await FirebaseAuth.instance.verifyPhoneNumber(
        phoneNumber: phoneNumber,
        verificationCompleted: (PhoneAuthCredential credential) async {
          await firebaseAuth.signInWithCredential(credential);
        },
        timeout: Duration(seconds: 60),
        verificationFailed: (FirebaseAuthException e) {
          print('Failed to verify phone number: ${e.message}');
        },
        codeSent: (String verificationId, int? resendToken) async {
          print('OTP sent: $verificationId');
          SignInPage.verify = verificationId;
          Get.off(OTPPage());
        },
        codeAutoRetrievalTimeout: (String verificationId) {
          print('OTP auto retrieval timeout: $verificationId');
        },
      );
    } catch (e) {
      print('Error sending OTP: $e');
    }
  }

### pubspec.yaml

`
dependencies:
 flutter:
    sdk: flutter
  cloud_firestore: ^4.4.5
  cupertino_icons: ^1.0.2
  firebase_auth: ^4.3.0
  firebase_core: ^2.8.0
  firebase_storage: ^11.0.16
  get: ^4.6.1
  google_fonts: ^4.0.3
  pinput: ^2.2.31
  play_integrity_flutter: ^0.0.1
`

**Comment by charleshoong97** [comment]
@MSatyam-Mishra have you solve this issue?

**Reply by MSatyam-Mishra** [reply]
Yes, by downgrading package version it started working perfectly.
